cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,73 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import homePageObject from '../support/pages/home.pageObject';
+import settingsPageObject from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-describe('Settings page', () => {
-  before(() => {
+const homePage = new homePageObject();
+const settingsPage = new settingsPageObject();
+const signInPage = new SignInPageObject();
 
-  });
+const userData = {
+  username: 'riot',
```

Comment:
Use 'faker' library to generate fake user data instead of hardcoding the values.